### PR TITLE
Create scope

### PR DESCRIPTION
**Pull request by carlaquinn**

### CODE REVIEW

**Review by celestehorgan — APPROVED** [review]
Tenatative LGTM, let's see what everyone else has to say though!

**Comment by LarryKunz** [comment]
I really like this scope document. It describes, clearly and succinctly, what I think most of us have been expressing. Thank you, Carla!

I wonder if we can think of positive examples for all of the categories, in addition to the negative examples. For example, for "gender identity," the use of "he" or "him" to refer to any person is an example. I can't think of an example for the homophobic category, but perhaps someone else can.

**Comment by carlaquinn** [comment]
Update to scope of terms statement: The initial scope addresses terms that reinforce systemic discrimination in which a dominant group prevents or inhibits the full social, economic, and political participation of other people based on their identity. This applies to terms that reflect systemic racism, as well as discrimination by sex, gender, ability, orientation, and any identity-based discrimination.

File: ws-language/scope [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+**Output:** An easily accessed and clear single source of truth with terms to avoid, use, or replace along with a set of best practices and principles.
+
+**Scope of terms:** The initial focus addresses terms that reinforce systemic discrimination in which a dominant group prevents or discourages the full social, economic, and political participation of other people based on their identity. This applies to terms that reflect systemic racism, as well as discrimination by sex, gender, and ability, and any other identity-based discrimination.
+
+These terms are described in the [LF framework](https://github.com/kubernetes/community/blob/master/wg-naming/language-evaluation-framework.md) as first order concerns:
+ - Is the term overtly racist?
+    Examples include “master/slave”.
+-   Is the term overtly sexist, transphobic, or pejorative about a gender identity?
+    Examples do not include “transclusion” of dependencies, or “binary” operators.
+-   Is the term overtly ableist, or pejorative to neurodiverse or disabled people
```
Comment (lex-joy) [inline ws-language/scope:18]: Missing question mark here. Otherwise, I give this my vote of approval!

**Comment by quaid** [comment]
> I really like this scope document. It describes, clearly and succinctly, what I think most of us have been expressing. Thank you, Carla!
> 
> I wonder if we can think of positive examples for all of the categories, in addition to the negative examples. For example, for "gender identity," the use of "he" or "him" to refer to any person is an example. I can't think of an example for the homophobic category, but perhaps someone else can.

I agree this looks good and understand what you are looking for, so I propose this: let's commit this document the way it is, then we can have a separate merge request to create the positive/negative balance.

**Review by celestehorgan — APPROVED** [review]

**Pull request merged**